### PR TITLE
avoid links and buttons being able to be switched

### PR DESCRIPTION
**Pull request by vramosp**

### CODE REVIEW

File: models/Link.yaml [hunk 1]
```diff
@@ -57,10 +57,6 @@ fields:
     options:
```
Comment (smnh) [inline models/Link.yaml:57]: if link cannot be changed, maybe it is better to remove this field altogether?

Comment (vramosp) [reply models/Link.yaml:57]: @ScottyMJacobson said he'll take care of cleaning it up :)

Comment (smnh) [reply models/Link.yaml:57]: cool

Comment (ScottyMJacobson) [reply models/Link.yaml]: @smnh how concerned are we about backwards-compatibility right now? will projects that have a `Link` with this `style` field break or just print an error?

Comment (JugglerX) [reply models/Link.yaml]: @ScottyMJacobson I tested this in Studio and style field was removed.

It seems like to release this safely the following things still need to be done.

* Can you please check the component itself and add the proper guards to link and button if the style prop is missing?
* Can you also update any content in `stackbit-content-packs` and remove the style field.

**Pull request merged**